pkg/api/kubegraph/edges.go

```diff
@@ -246,3 +249,34 @@ func AddHPAScaleRefEdges(g osgraph.Graph) {
 		g.AddEdge(hpaNode, syntheticNode, ScalingEdgeKind)
 	}
 }
+
+func BelongsToDeployment(d *extensions.Deployment, rs *extensions.ReplicaSet) bool {
```

Comment:
again something that is already public in upstream but not here... I will add todo.